runtime/lua/vim/lsp/rpc.lua

```diff
@@ -294,7 +392,14 @@ local function create_and_start_client(cmd, cmd_args, handlers, extra_spawn_para
     }
   end
 
-  local function send_request(method, params, callback)
+  -- FIXME: Should be placed on the RPC client object returned by `start()`
```

Comment:
Yeah, some of this I hope to one day handle with tree sitter generated docs...

Reply:
I am anxiously awaiting this, yeah :XD: